File: v03/zad2/Krug.cpp
#include "Krug.hpp"

// Podrazumevani konstruktor
Krug::Krug(double r) {
    this->r = r;
}

// Vraća poluprečnik kruga
double Krug::getR() const {
    return r;
}

// Vraća obim kruga
double Krug::getO() const {
    return 2 * r * M_PI;
}

// Vraća površinu kruga
double Krug::getP() const {
    return pow(r, 2) * M_PI;
}

File: v10/zad2/Krug.java
package zad2;

public class Krug {
	
	// Polja
	private double r;

	public Krug(double r) {
		super();
		this.r = r;
	}
	
	// Geteri i seteri (izgenerisani)
	public double getR() {
		return r;
	}

	public void setR(double r) {
		this.r = r;
	}
	
	// Metoda koja vraća obim
	public double obim() {
		return 2 * r * Math.PI;
	}
	
	// Metoda koja vraća površinu
	public double povrsina() {
		return Math.pow(r, 2) * Math.PI;
	}
	
}
